include/reactor-uc/connection.h

```diff
@@ -65,19 +39,13 @@ void LogicalConnection_ctor(LogicalConnection *self, Reactor *parent, Port **dow
 struct DelayedConnection {
   Connection super;
   interval_t delay;
-  TriggerDataQueue trigger_data_queue;
+  bool is_physical;
```

Comment:
is this for expressing DelayedPhysical Connections or is this redundant because the flag already exists in the Action?

Reply:
Here I am merging delayed and physical connection into a single type. DelayedConnection, so it can have a delay and be physical. It does not inherit from Aciton, rather it is a separate kind of trigger

Reply:
How to we express a normal PhysicalConnection then?

This call `is_physical=true` and `delay = 0` ?

Reply:
Yes, or rather, delay=NEVER, delay=0 means a microstep delay

Reply:
We should make a separate macro for physical connections also